resources/options_lookup.tsv

```diff
@@ -1289,11 +1289,11 @@ Clothes Washer Presence	Yes	ResStockArguments	clothes_washer_present=true
 Clothes Washer Usage Level	100% Usage	ResStockArguments	clothes_washer_usage_multiplier=1.0
 Clothes Washer Usage Level	120% Usage	ResStockArguments	clothes_washer_usage_multiplier=1.2
 Clothes Washer Usage Level	80% Usage	ResStockArguments	clothes_washer_usage_multiplier=0.8
-Cooking Range	Electric Induction	ResStockArguments	cooking_range_oven_present=true	cooking_range_oven_location=auto	cooking_range_oven_fuel_type=electricity	cooking_range_oven_is_induction=true	cooking_range_oven_is_convection=auto
-Cooking Range	Electric Resistance	ResStockArguments	cooking_range_oven_present=true	cooking_range_oven_location=auto	cooking_range_oven_fuel_type=electricity	cooking_range_oven_is_induction=false	cooking_range_oven_is_convection=auto
-Cooking Range	Gas	ResStockArguments	cooking_range_oven_present=true	cooking_range_oven_location=auto	cooking_range_oven_fuel_type=natural gas	cooking_range_oven_is_induction=false	cooking_range_oven_is_convection=auto
-Cooking Range	None	ResStockArguments	cooking_range_oven_present=false	cooking_range_oven_location=auto	cooking_range_oven_fuel_type=natural gas	cooking_range_oven_is_induction=false	cooking_range_oven_is_convection=auto
-Cooking Range	Propane	ResStockArguments	cooking_range_oven_present=true	cooking_range_oven_location=auto	cooking_range_oven_fuel_type=propane	cooking_range_oven_is_induction=false	cooking_range_oven_is_convection=auto
+Cooking Range	Electric Induction	ResStockArguments	cooking_range_oven_present=true	cooking_range_oven_location=auto	cooking_range_oven_fuel_type=electricity	cooking_range_oven_is_induction=true	cooking_range_oven_is_convection=auto	electric_panel_load_cooking_range_power=10000
```

Comment:
Where did the 10000 W value come from? The [project spreadsheet](https://github.com/NREL/resstock/blob/electrical_panels/Electrical_Panels_EUSS/panels/nameplate_rating_new_load.csv) uses 12,000 W for a 240V induction range